src/utility/include/mp-units/math.h

```diff
@@ -136,6 +136,36 @@ template<auto R, typename Rep>
   return {static_cast<Rep>(abs(q.numerical_value_ref_in(q.unit))), R};
 }
 
+/**
+ * @brief Computes the fma of 3 quantities
+ *
+ * @param a: Multiplicand
+ * @param x: Multiplicand
+ * @param b: Addend
+ * @return Quantity: The nearest floating point representable to ax+b
+ */
+template<auto R, auto S, auto T, typename Rep>
+[[nodiscard]] constexpr quantity<common_reference(R* S, T), Rep> fma(const quantity<R, Rep>& a,
+                                                                     const quantity<S, Rep>& x,
+                                                                     const quantity<T, Rep>& b) noexcept
```

Comment:
I think you allow the representation types to be heterogeneous like I suggest.
That works with `std::fma` (https://cpp1.godbolt.org/z/KjY3efnYY): `fma(2 * m, 3.0f * m, 1.0 * m2)`.